atlas/commands/bootstrap/templates/cmd/server/grpc.go.gotmpl

```diff
@@ -16,6 +15,9 @@ func NewGRPCServer(logger *logrus.Logger{{ if .WithDatabase }}, dbConnectionStri
 	grpcServer := grpc.NewServer(
 	grpc.UnaryInterceptor(
 		grpc_middleware.ChainUnaryServer(
+			// Request-Id interceptor
+			//requestid.UnaryServerInterceptor(),
```

Comment:
This interceptor should be the first one. Also this line causes the problem with dep init.
Without this line new project is inited with latest atlas-app-toolkit release (v0.12.2 at the moment).
If this line is added after `dep init` the project will reference atlas-app-toolkit v0.5.0 which is wrong.

Reply:
I plan to create a separate bug and investigate this problem in scope of this bug. For now the root cause is not clear unless you have any ideas.

Reply:
requestid interceptor should be after logging interceptor so that requestid generated/already present can be added to logging context

Reply:
requestid interceptor should be after logging interceptor so that requestid generated/already present can be added to logging context

Reply:
@kumaya, it is actually vice versa, it should be before other interceptors that need resource id. Consider following code sample:
```
func Interceptor1 {
    fmt.Println("1 before");
    handler(...)
    fmt.Println("1 after");
}

func Interceptor2 {
    fmt.Println("2 before");
    handler(...)
    fmt.Println("2 after");
}

func main() {
.....
	interceptors := []grpc.UnaryServerInterceptor{
		Interceptor1
		Interceptor2
	}
.....
}
```

Code above will print out:
```
1 before
2 before
2 after
1 after
```

Reply:
Looking at [contacts app](https://github.com/Infoblox-CTO/atlas.contacts.app/blob/master/cmd/server/grpc.go#L28) I see that logging interceptor is before requestid. The reason it works is that generation of request id happens before handler is called while logging logic is after handler call.

Reply:
@Evgeniy-L requestid interceptor by design adds generated/passed request-id to logging context. Unless logging fields exists in context, not sure about the behavior.
```
func addRequestIDToLogger(ctx context.Context, reqID string) {
	ctxlogrus.AddFields(ctx, logrus.Fields{DefaultRequestIDKey: reqID})
}
```

```
reqID := handleRequestID(ctx)
// add request id to logger
addRequestIDToLogger(ctx, reqID)
```

Reply:
@kumaya, I agree that requestid interceptor is important and should be included in cli. This is why I created a separate issue to return it back once the issue with `dep init` is resolved.